pyccl/cosmology.py

```diff
@@ -156,84 +156,84 @@ class Cosmology(CCLObject):
-        w0 (:obj:`float`, optional): First order term of dark energy equation
+        w0 (:obj:`float`): First order term of dark energy equation
             of state. Defaults to -1.
-        wa (:obj:`float`, optional): Second order term of dark energy equation
+        wa (:obj:`float`): Second order term of dark energy equation
             of state. Defaults to 0.
         T_CMB (:obj:`float`): The CMB temperature today. The default of
             is 2.725.
-        bcm_log10Mc (:obj:`float`, optional): One of the parameters of the
+        bcm_log10Mc (:obj:`float`): One of the parameters of the
             BCM model. Deprecated.
-        bcm_etab (:obj:`float`, optional): One of the parameters of the BCM
+        bcm_etab (:obj:`float`): One of the parameters of the BCM
             model. Deprecated.
-        bcm_ks (:obj:`float`, optional): One of the parameters of the BCM
+        bcm_ks (:obj:`float`): One of the parameters of the BCM
             model. Deprecated.
-        mu_0 (:obj:`float`, optional): One of the parameters of the mu-Sigma
+        mu_0 (:obj:`float`): One of the parameters of the mu-Sigma
             modified gravity model. Defaults to 0.0
-        sigma_0 (:obj:`float`, optional): One of the parameters of the mu-Sigma
+        sigma_0 (:obj:`float`): One of the parameters of the mu-Sigma
             modified gravity model. Defaults to 0.0
-        c1_mg (:obj:`float`, optional): MG parameter that enters in the scale
+        c1_mg (:obj:`float`): MG parameter that enters in the scale
             dependence of mu affecting its large scale behavior. Default to 1.
             See, e.g., Eqs. (46) in Ade et al. 2015, arXiv:1502.01590
             where their f1 and f2 functions are set equal to the commonly used
             ratio of dark energy density parameter at scale factor a over
             the dark energy density parameter today
-        c2_mg (:obj:`float`, optional): MG parameter that enters in the scale
+        c2_mg (:obj:`float`): MG parameter that enters in the scale
             dependence of Sigma affecting its large scale behavior. Default 1.
             See, e.g., Eqs. (47) in Ade et al. 2015, arXiv:1502.01590
             where their f1 and f2 functions are set equal to the commonly used
             ratio of dark energy density parameter at scale factor a over
             the dark energy density parameter today
-        lambda_mg (:obj:`float`, optional): MG parameter that sets the start
+        lambda_mg (:obj:`float`): MG parameter that sets the start
             of dependance on c1 and c2 MG parameters. Defaults to 0.0
             See, e.g., Eqs. (46) & (47) in Ade et al. 2015, arXiv:1502.01590
             where their f1 and f2 functions are set equal to the commonly used
             ratio of dark energy density parameter at scale factor a over
             the dark energy density parameter today
-        df_mg (array_like, optional): Perturbations to the GR growth rate as
```

Comment:
Why did you remove the "optional" information?

Reply:
This is just spurious. The docs already show which arguments have default values and which ones need to be specified.